README.md

```diff
@@ -31,9 +45,26 @@ or
 ```
 rails s 
 ```
+
+## Functionality
```

Comment:
I think this is supposed to be more akin to our functionality report. I added in a github-formatted version as a pull request to your repository, if you want to merge that in?